components/checkbox/checkbox-group.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div :class="prefixCls">
+    <slot>
+      <checkbox v-for="(item,index) in items" :key="index"
```

Comment:
1. Attribute "bind" should be on a new line. Similarly hereinafter.
2. About indent cloud reference [Button](https://github.com/zzuu666/antue/blob/master/components/button/button.vue).